docs/user-questions.rst

```diff
@@ -112,12 +111,13 @@ and then convert them to a :class:`pint.toa.TOAs` object::
     t = pulsar_mjd.Time(np.array([55000, 56000]), scale="utc", format="pulsar_mjd")
     obs = "gbt"
 
-    toalist = [toa.TOA(tt, obs=obs, error=1 * u.us) for tt in t]
-    toas = toa.get_TOAs_list(toalist)
+    toas = toa.get_TOAs_array(t, obs)
 
 Note that we also use ``Time`` from :mod:`pint.pulsar_mjd` rather than :class:`astropy.time.Time` directly to allow the 
 ``pulsar_mjd`` format, designed to avoid leap seconds.  But this would work with standard :class:`astropy.time.Time` 
```

Comment:
I wouldn't say that pulsar_mjd "avoids" leap seconds, but rather that is handles leap seconds according to standard conventions used in pulsar work. Also, I don't think you really need to use `pulsar_mjd.Time`. Once you have imported the code that defines `pulsar_mjd`, I think normal `Time` should work fine.

Reply:
OK, I have clarified that.